python/acl_anthology/collections/paper.py

```diff
@@ -250,6 +331,45 @@ def to_bibtex(self, with_abstract: bool = False) -> str:
             bibtex_fields.append(("abstract", self.abstract))
         return make_bibtex_entry(self.bibtype, self.bibkey, bibtex_fields)
 
+    def to_citation(self, style: Optional[str] = None) -> str:
+        """Generate a citation (reference) for this paper.
+
+        Arguments:
+            style: A path to a CSL file.  If None (default), uses the built-in ACL citation style.
```

Comment:
For python internal documentation purposes it would be nice to list the valid arguments here.

Reply:
The valid arguments are ... paths to a CSL file, like it says.

The Python library that provides a convenient interface to the official CSL style file repo has not been updated in forever and isn't working anymore with newer Python versions, so there is no option to provide _names_ of citation styles.